scripts/globals/conquest.lua

```diff
@@ -3,1388 +3,1360 @@
 --     Functions for Conquest system
 --
 -----------------------------------
-
-require("scripts/globals/keyitems");
-require("scripts/globals/missions");
-require("scripts/globals/common");
-
------------------------------------
--- convenience constants
------------------------------------
-
-NATION_SANDORIA = 0;
-NATION_BASTOK   = 1;
-NATION_WINDURST = 2;
-BEASTMEN        = 3;
-OTHER           = 4;
-
-RONFAURE        = 0;
-ZULKHEIM        = 1;
-NORVALLEN       = 2;
-GUSTABERG       = 3;
-DERFLAND        = 4;
-SARUTABARUTA    = 5;
-KOLSHUSHU       = 6;
-ARAGONEU        = 7;
-FAUREGANDI      = 8;
-VALDEAUNIA      = 9;
-QUFIMISLAND     = 10;
-LITELOR         = 11;
-KUZOTZ          = 12;
-VOLLBOW         = 13;
-ELSHIMOLOWLANDS = 14;
-ELSHIMOUPLANDS  = 15;
-TULIA           = 16;
-MOVALPOLOS      = 17;
-TAVNAZIANARCH   = 18;
-
-CONQUEST_TALLY_START = 0;
-CONQUEST_TALLY_END   = 1;
-CONQUEST_UPDATE      = 2;
-
-nationAlly = 3;
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+require("scripts/globals/common")
+
+dsp = dsp or {}
+dsp.conquest = {}
+
+dsp.conquest.NATION_SANDORIA = 0
```

Comment:
Should this be divided into dsp.nation, dsp.region, and dsp.conquest? You assign the same values for dsp.conquest.CONQUEST_TALLY down below and that is super gross.

Also, assuming dsp.nation/dsp.region/dsp.conquest is defined in this file, just do dsp.nation= { NATION_SANDORIA = 0, NATION_BASTOK = 1, etc. }
More C-style enum and consistent with other tabled values.

You can still continue to define your functions down below outside the initializations, too.